src/Impostor.Server/Net/Inner/Objects/GameManager/InnerGameManager.cs

```diff
@@ -39,6 +39,21 @@ protected T AddGameLogic<T>(T logic)
         return logic;
     }
 
+    internal int? GetGameLogicTag<T>(T logic)
+        where T : GameLogicComponent
+    {
+        for (var i = 0; i < LogicComponents.Count; i++)
+        {
+            var component = LogicComponents[i];
+            if (component == logic)
+            {
+                return i;
+            }
+        }
+
+        return null;
```

Comment:
`return LogicComponents.IndexOf(logic)`?

Reply:
Could do that, did it like this because I maybe wanted to check types and base types to fetch a tag. So instead of `GetGameLogicTag(someInstance)` you would do `GetGameLogicTag<LogicOptions>()`, that way you don't need an instance. What is your opinion on that?